admin/app/components/target-profiles/badge-form.js

```diff
@@ -60,21 +70,49 @@ export default class BadgeForm extends Component {
     }
   }
 
-  async _createThresholdBadgeCriterion(badgeId) {
+  async _createThresholdBadgeCriterion(badge) {
     try {
       if (this.threshold < 0 || this.threshold > 100) {
         this.notifications.error('Le taux de réussite doit être compris entre 0 et 100.');
         return;
       }
       const badgeCriterion = this.store.createRecord('badge-criterion', {
         scope: 'CampaignParticipation',
-        threshold: this.threshold,
+        threshold: this.campaignParticipationThreshold,
+        badge,
       });
-      await badgeCriterion.save({ adapterOptions: { badgeId } });
+      await badgeCriterion.save();
       this.notifications.success('Le critère du résultat thématique a été créé.');
     } catch (error) {
       console.error(error);
       this.notifications.error('Erreur lors de la création du critère du résultat thématique.');
     }
   }
+
+  async _createSkillSetAndBadgeCriterion(badge) {
+    try {
+      if (this.skillSet.threshold < 0 || this.skillSet.threshold > 100) {
+        this.notifications.error('Le taux de réussite doit être compris entre 0 et 100.');
+        return;
+      }
+      const skillIds = this.skillSet.skills.split(',');
```

Comment:
```suggestion
      const skillIds = this.skillSet.skills.replace(/ /g,'').split(',');
```
J'ai testé en c/c des id, j'ai laissé des espaces, et ca a un peu cassés ma listes. Je propose donc cette petite modification